scripts/Package.hs

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env stack
+-- stack --resolver lts-12.12 script --package Cabal,directory,extra,filepath,Glob,process,temporary
+
+{-# LANGUAGE LambdaCase #-}
+
+import Control.Monad
+import Control.Monad.Extra
+import Data.Maybe
+import Data.Monoid
+import Distribution.Simple.Program.Find
+import Distribution.Simple.Utils
+import Distribution.Verbosity
+import System.Directory
+import System.Environment
+import System.FilePath
+import System.FilePath.Glob
+import System.IO.Temp
+import System.Process
+
+depsArchiveUrl :: String
+depsArchiveUrl = "https://s3-us-west-2.amazonaws.com/packages-luna/dataframes/libs-dev-v140.7z"
+
+packageBaseUrl :: String
+packageBaseUrl = "https://s3-us-west-2.amazonaws.com/packages-luna/dataframes/windows-package-base.7z"
+
+getEnvDefault :: String -> String -> IO String
+getEnvDefault variableName defaultValue =
+    fromMaybe defaultValue <$> lookupEnv variableName
+
+findProgram :: ProgramSearchPath -> String -> IO (Maybe FilePath)
+findProgram whereToSearch name = do
+    fmap fst <$> findProgramOnSearchPath silent whereToSearch name
+
+download :: String -> FilePath -> IO ()
+download url destPath = callProcess "curl" ["-fSL", "-o", destPath, url]
+
+find7z :: IO (Maybe FilePath)
+find7z = do
+    let default7zPath = "C:\\Program Files\\7-Zip"
```

Comment:
extract this magic paths somehere

Reply:
It is already extracted to a binding of its own, I don't see really any value in moving this to the global scope. 

But I want to refactor the program finding/calling in the next iteration, as there is some code pattern duplication now and it would only get worse with all the programs we will need to add to support other platforms. When refactoring a proper place for providing such defaults should be decided as well.